apps/judicial-system/api/infra/judicial-system-api.ts

```diff
@@ -35,8 +35,8 @@ export const serviceSetup = (services: {
       },
       HIDDEN_FEATURES: {
         dev: '',
-        staging: 'INDICTMENTS',
-        prod: 'INDICTMENTS',
+        staging: 'INDICTMENTS, INDICTMENT_ROUTE',
```

Comment:
Looking at the DSL, I cant see how the comma separated string will be parsed. Is it guaranteed that it will work as expected?